transformations/disability_transformation/transformation.py

```diff
@@ -0,0 +1,213 @@
+!python -m spacy download en_core_web_sm
+
+import spacy
+import string
+from spacy.lang.en.examples import sentences 
+nlp = spacy.load("en_core_web_sm")
+
+
+
+
+# A dict containing offensive words and their alternatives
+disability_names =  {"blind":"person or people with a visual impairment",
```

Comment:
I wonder if it would be a good idea to move this dict into an external json file so it can be edited separately? What do you think @kaustubhdhole?

Reply:
The dictionary is small anyways so would be fine both ways.